src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -470,7 +470,7 @@ public function getEvent()
     public function hasPermission()
     {
         if (!$this->parameters->has('permission')) {
-            return true;
+            return false;
         }
 
         return false !== $this->parameters->get('permission');
```

Comment:
Why can't we define some default value as a second parameter? Won't it be equal?
```php
if (!$this->parameters->has('permission')) {
    return false;
}
 
return false !== $this->parameters->get('permission');
```
/\
||
\/
```php
return false !== $this->parameters->get('permission', false);
```